createsend/createsend.py

```diff
@@ -144,12 +142,24 @@ def make_request(self, method, path, params={}, body="", username=None,
       # Check that the actual url which would be requested matches self.faker.url.
       actual_url = "https://%s%s" % (parsed_base_uri.netloc, self.build_url(parsed_base_uri, path, params))
       self.faker.actual_url = actual_url
-      if self.faker.url != actual_url:
```

Comment:
Because hash randomization is enabled by default starting from 3.3, comparing the URL strings doesn't work in tests because the ordering in the query string can be different each time.